funk.py

```diff
@@ -0,0 +1,58 @@
+def numbSearch(minN, maxN):
```

Comment:
use shake case for functions and variables name
```suggestion
def numb_search(min_n, max_n):
```